astropy/modeling/functional_models.py

```diff
@@ -349,6 +354,17 @@ def __init__(self, amplitude=amplitude.default, x_mean=x_mean.default,
                 y_vec = eig_vecs[:, 0]
                 theta = np.arctan2(y_vec[1], y_vec[0])
 
+        # Ensure stddev makes sense if its bounds are not explicitly set.
+        # stddev must be non-zero and positive.
+        if 'bounds' not in kwargs:
+            kwargs['bounds'] = {'x_stddev': (FLOAT_EPSILON, None),
+                                'y_stddev': (FLOAT_EPSILON, None)}
+        else:
+            kwargs['bounds']['x_stddev'] = kwargs['bounds'].get(
+                'x_stddev', (FLOAT_EPSILON, None))
+            kwargs['bounds']['y_stddev'] = kwargs['bounds'].get(
+                'y_stddev', (FLOAT_EPSILON, None))
+
```

Comment:
I think because of `Gaussian2D` re-defining the `__init__`? I am just guessing here. Convolution tests failed horribly when I put `bounds` in `Parameter`, to the point the tests just hang until I hit CTRL+C. Maybe @nden can explain.